core/storage/topic/gae_models.py

```diff
@@ -422,9 +425,46 @@ def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
             'thumbnail_filename': base_models.EXPORT_POLICY.NOT_APPLICABLE,
             'thumbnail_bg_color': base_models.EXPORT_POLICY.NOT_APPLICABLE,
             'version': base_models.EXPORT_POLICY.NOT_APPLICABLE,
-            'url_fragment': base_models.EXPORT_POLICY.NOT_APPLICABLE
+            'url_fragment': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'story_exploration_mapping':
+                base_models.EXPORT_POLICY.NOT_APPLICABLE
         })
 
+    @classmethod
+    def get_story_exploration_mappings(cls) -> List[Dict[str, List[str]]]:
+        """Gets each entity's story_exploration_mapping property.
+
+        Returns:
+            list(dict(str, list(str))). A list of
+            story_exploration_mapping values stored in each entity.
+        """
+        projections: Sequence[TopicSummaryModel] = cls.query(
+            projection=['story_exploration_mapping']
+        ).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
+        return [
+            projection.story_exploration_mapping for projection in projections
+        ]
+
+    @classmethod
+    def get_story_exploration_mappings_by_name(
+        cls, name: str
```

Comment:
Don't do this. Drop the method; have the caller get the topic ID, fetch the summary by ID, and read its story_exploration_mappings property.

(Also for future reference, args like this should be called topic_name instead.)